README.md

```diff
@@ -15,7 +15,7 @@ Additional description about the project and its features.
 - Technologies used: **FIGMA**, **ESLINT**, **STYLELINT**, and **GitHub CI/CD**
 
 ## Live Demo
-
+[Live Video Link](https://www.loom.com/share/6338e379f940419db989b1d291fb276d)
```

Comment:
- [ ] Thank you for the project presentation. It will be nice to add a highlight about a piece of the code you find interesting or is proud of making in your next presentation.

Reply:
I just checked again. You need to show us a piece code you find interesting in the presentation and why you are proud of it. All your presentation is only on the UI. I didn't see any code...